guides/deploy/README.md

```diff
@@ -371,6 +371,107 @@ akash \
 
 where `$SERVICE_NAME` is the name of a service defined in your [SDL](/sdl).
 
+## Update your deployment
+
+Updating active deployments is a two step process. First, edit the deploy.yml with the desired changes.
```

Comment:
I would change this to be a bit more general? Like "edit your deployment YAML file"

Reply:
agree. fixed